embeddings/embedding/static/magnitude.py

```diff
@@ -0,0 +1,49 @@
+import hashlib
+import os
+from dataclasses import dataclass
+from pathlib import Path
+from typing import List, cast
+from urllib.parse import urlparse
+
+import appdirs
+import requests
+from numpy import ndarray
+from pymagnitude import Magnitude
+
+from embeddings.embedding.embedding import Embedding
+from embeddings.utils.loggers import get_logger
+
+_logger = get_logger(__name__)
+
+
+@dataclass
+class MagnitudeConfig:
+    model_path: str
+
+
+class MagnitudeEmbedding(Embedding[List[str], ndarray]):
+    def __init__(self, config: MagnitudeConfig) -> None:
+        self.model = Magnitude(config.model_path)
+
+    def embed(self, data: List[str]) -> ndarray:
```

Comment:
@riomus it would be great to add magnitude that will work with our pipelines. For now, it would be in the repo, but not used ;p can you check out how hard it would be to add FlairEmbedding using magnitude?

Reply:
added flair wrapper for magnitude